### ISSUE TITLE
less style do not use variables.less correctly

### ISSUE THREAD

**Issue by genuinefafa** [uxsolutions/bootstrap-datepicker#712]
I noticed that the [less file](https://github.com/eternicode/bootstrap-datepicker/blob/master/less/datepicker.less) is forcing certain colors, therefor if you choose to modify bootstrap to suit your needs, it breaks.

Any idea on how to fix this?

**Comment by GabrielGil** [comment]
Two years ago.. Is there any advance with this?Anybody making any work on it?

**Comment by genuinefafa** [comment]
@GabrielGil it's being 17 months, less than a year and a half :grin: 

Anyways, I've tried without luck. There are some magic colors going on there that, in my case, was better tune manually, sorry. Maybe I was not as experience as I thought I was. If you wanna do it youself I'm sure @eternicode, me and everyone else will be glad! :smiley:

**Comment by acrobat** [comment]
Yes the theming feature is something I also want to get in the datepicker, but it's a big piece of work, therefor we are now putting more work in all the smaller features and many bugs to get them out as fast as possible. My idea is that theming is something we can tackle in a next major version (2.0)

But feel free to start working on this feature, I will try to give as much support as possible!

**Comment by GabrielGil** [comment]
hahaha, you are definitely right. My apologies @genuinefafa :) 
I'd love to give it a look, but i think is better to pair up sass (#1042) and less versions, since if they are not too complex can be maintained simultaneously without that much of a hassle.

About the theming @acrobat , I'm currently integrating the datepicker in a project, so I think I'm gonna get familiar with all the markup and current structure, so i'd be up for helping on creating/improving this feature.

Thank you both for such fast replies! When I hit submit thought somebody will answer later than sooner.

**Comment by roooodcastro** [comment]
Is there any chance this might get picked up again?

IMO we don't need a full theming feature, the simple ability to change the default colors is more than enough. Especially with Bootstrap, not everyone uses the default Bootstrap colors, so this plugin feels out of place and due to the way the LESS file is compiles do CSS, is very troublesome to manually change all the colors.

I think it's a simple matter of setting some variables and using it, so the developer will be able to override it.

I also use SASS, so #1042 is really important for this as well.

And in case 2.0.0 is still far into the future (there isn't even a 2.0 branch yet), is it possible to include it in 1.x? This gem is perfect for Bootstrap, but this is killing me ):

**Comment by acrobat** [comment]
I've been quite busy last months but I'm trying to make some time soon and have the 1.7 release out. This should be the last minor release after that we can start working on 2.0 and the theming support. When we have a 2.0 branch available I will ping this issue again so people who are interested in the theming feature can help implementing this.
